packages/@aws-cdk/aws-service-spec/build/patches/service-patches/core.ts

```diff
@@ -201,6 +201,32 @@ export namespace fp {
     );
   }
 
+  type SchemaType = "string" | "number" | "boolean" | "object" | "array" | "integer" | "null";
+
+  /**
+   * Add properties to `properties`
+   */
+  export function addPropertys<TypeName extends string = string>(
```

Comment:
is it mandatory to use generics here .. I think the resource type parameter will be always string